deploy/operator.yaml

```diff
@@ -650,7 +650,7 @@ spec:
     protocol: TCP
     targetPort: 9443
   selector:
-    name: ssp-operator
+    control-plane: ssp-operator
```

Comment:
This file is autogenerated, so fixing it inline won't do - probably need to fix this at the manifest-templator.

Reply:
done